plugins/mcreator-localization/lang/texts.properties

```diff
@@ -907,6 +907,7 @@ blockly.block.entity_getridingentity=%2 Get entity %1 is riding
 blockly.block.entity_gettargetentity=%2 Get entity %1 is targeting for attack
 blockly.block.entity_has_achievement=Has %1 completed advancement %2
 blockly.block.entity_has_recipe=Has %1 unlocked recipe %2
+blockly.block.entity_has_screen_open=Does %1 have an open GUI:
```

Comment:
Maybe you should write differently like "Is currently opened GUI of %1 of type: ". It might be more understandable for beginners.